jcommune-view/jcommune-web-view/src/main/webapp/resources/css/app/application.css

```diff
@@ -598,6 +598,12 @@ div.signup {
     font-style: italic;
 }
 
+.topic-draft-counter {
```

Comment:
I dunno what exactly is the reason, but with new changes the space between topic subject and Sticked checkbox increased.